_docs-2/administration/upgrading.md

```diff
@@ -38,6 +56,43 @@ will then need to transfer the contents of the current individual files to this
 **Warning**: Upgrading a previously encrypted instance is not supported as the implementation
 and properties have changed.
 
+## Upgrading from 2.0 to 2.1
+
+Upgrading to Accumulo 2.1 is done by stopping Accumulo 2.0 and starting Accumulo 2.1.
+Before stopping Accumulo 2.0, install Accumulo 2.1 and configure it by following the 
+[Quick Start]({% durl getting-started/quickstart %}).
+
+Stop Accumulo.
+(optional) create a ZooKeeper snapshot using:
+
+`accumulo dump-zoo --xml --root /accumulo | tee PATH_TO_SNAPSHOT` 
```

Comment:
This inline code will look funny on its own. If you want a code block, use triple quotes:

````suggestion
```
accumulo dump-zoo --xml --root /accumulo | tee PATH_TO_SNAPSHOT
```
````